extensions/metrics/src/__examples__/demo.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# This script runs prometheus to scape and visulize metrics collected for the
+# example LoopBack 4 application.
```

Comment:
Have you considered promoting the example into a regular package in `examples/` directory, e.g. `examples/prometheus-metrics`?